opm/autodiff/FullyImplicitBlackoilSolver.hpp

```diff
@@ -351,6 +353,17 @@ namespace Opm {
         /// residual mass balance (tol_cnv).
         bool getConvergence(const double dt, const int iteration);
 
+        /// \brief Compute the reduction within the convergence check.
+        /// \return The total pore volume.
+        double
```

Comment:
I'd like at least a comment explaining that expected (necessary) sizes of the input and output arguments (n by 3 and 1 by 3 I think?) since I misunderstood this at first...